src/frontend/app/components/BrowseScreenSearchWidget.tsx

```diff
@@ -131,81 +131,79 @@ const SearchResults = ({
   }, [items]);
 
   return (
-    <>
-      <div ref={containerRef}>
-        <div className="flex flex-row flex-wrap gap-5 mb-8">
-          <div className="relative max-w-96 w-full">
-            <input
-              id="search-input"
-              className="w-full rounded-full pl-5 pr-10 pt-1 h-10 shadow-element cursor-pointer focus:outline-none placeholder:text-black-light"
-              type="text"
-              defaultValue={q}
-              placeholder="Search"
-              onFocus={() => setIsSearchWidgetVisible(true)}
-              onKeyUp={handleKeyUp}
-              onKeyDown={handleKeyDown}
-              autoComplete="off"
-            />
-            <FontAwesomeIcon icon={faSearch} className="absolute h-4 right-4 bottom-3" />
-          </div>
-          {selectedFacets.length > 0 && (
-            <div className="flex flex-row flex-wrap gap-2 items-center">
-              {selectedFacets.map((facet, index) => (
-                <FacetValue
-                  key={index}
-                  facetId={facet.facetId}
-                  facetValueId={facet.facetValueId}
-                  type="valueId"
-                  facetValueLabel={facet.valueLabel ?? ''}
-                  showRemoveIcon
-                  onRemoveIconClick={onRemoveFilter}
-                />
-              ))}
-              <button className="ml-3 text-violet-dark" onClick={onClearFilters}>
-                Clear all
-              </button>
-            </div>
-          )}
+    <div ref={containerRef} className="relative">
```

Comment:
Minor note, now the selected facets are hidden by the widget 
<img width="571" alt="image" src="https://github.com/Sitecore/Sitecore.Demo.Scenarios/assets/96472119/1a00a01b-a8ed-4bea-81b6-d5325acfe565">

Reply:
Yes, not ideal, but if we don't do that there could be a point where the dropdown is too short (when multiple facets are selected). So I think, since the selected facets are marked in violet in the dropdown, it's clear enough what is selected when the dropdown is open. The only functionality that is a bit compromised is the "Clear all" which is not that essential in this view.